### ISSUE TITLE
Support macOS 10.15+

### ISSUE THREAD

**Issue by ywu2014** [rainyl/opencv_dart#249]
opencv_dart version is opencv_dart: ^1.2.4, but when i start app using command
`flutter run -d macos`
it shows Failed to load dynamic library 'libopencv_dart.dylib': dlopen failed: library "libopencv_dart.dylib" not found
the readme document said it will download libs auto automatically, but it didn't fot me.

so i manually run commands below
```
flutter pub add opencv_dart
dart run opencv_dart:setup macos --arch x64
```
and it did download libs，but still show "libopencv_dart.dylib" not found

then i set the lib environment below
```
export OPENCV_DART_DISABLE_AUTO_BUILD=1
export OPENCV_DART_LIB_PATH=/Users/ywu/.pub-cache/hosted/pub.flutter-io.cn/opencv_dart-1.2.4/macos/GNUSparseFile.0/libopencv_dart.dylib
```
and this time，it shows no suitable image found
<img width="708" alt="image" src="https://github.com/user-attachments/assets/2e9e7e2f-86e8-44f8-9723-c7aba4b29241">

my os version is macOS Catalina 10.15.7

**Comment by rainyl** [comment]
@ywu2014 The minimum supported macos version is 11.0, I will try to support 10.15, you can try to build the project by yourself for now.

**Comment by ywu2014** [comment]
thx，then about what time will you support 10.15，is there a general plan？

**Comment by rainyl** [comment]
I am migrating the entire build system to cmake, including opencv.full, the most important dependency is ffmpeg, since github doesnt provide macos 10.15 runner, ffmpeg needs to be properly configured, also I am planning to support ffmpeg on android and ios, so it may take a long time to finish.

**Comment by ywu2014** [comment]
@rainyl  OK，thx！

**Comment by rainyl** [comment]
Never mind.

I am going to reopen this issue to remind myself, hope you wont mind it.

**Comment by ywu2014** [comment]
No problem!

**Comment by rainyl** [comment]
opencv_dart and opencv_core version v1.3.0 have been published, with macos 10.15 and above supported.
